python/Produce_Config_File.py

```diff
@@ -0,0 +1,136 @@
+import xlrd
+from array import array
+import numpy as np
+import ROOT as r
+from PlotOptions import parser
+import sys
+
+if len(sys.argv)==1:
+    parser.print_help()
+    sys.exit(1)
+    pass
+
+def CyclicColor(DrawArg):
+	if DrawArg % 7 == 0:
+		ret_int='kRed'
+		pass
+	elif DrawArg % 7 == 1:
+		ret_int='kRed+1'
+		pass
+	elif DrawArg % 7 == 2:
+		ret_int='kRed+2'
+		pass
+	elif DrawArg % 7 == 3:
+		ret_int='kRed+3'
+		pass
+	elif DrawArg % 7 == 4:
+		ret_int='kBlue'
+		pass
+	elif DrawArg % 7 == 5:
+		ret_int='kBlue+1'
+		pass
+	elif DrawArg % 7 == 6:
+		ret_int='kBlue+2'
+		pass
+	return ret_int
+
+(args,opts)=parser.parse_args()
+text_file = open(args.OutputName+".cfg", "w")
+
+DrawArg=0
+
+text_file.write("[BEGIN_CANVAS]\n")
+text_file.write("\tCanv_Axis_NDiv = '%s';#X,Y\n"%(args.AxisNDiv))
+text_file.write("\tCanv_Dim= '%s';#X,Y\n"%(args.CanvDim))
+text_file.write("\tCanv_DrawOpt = '%s';\n"%(args.CanvDrawOpt))
+text_file.write("\tCanv_Grid_XY = '%s';\n"%(args.CanvGridXY))
+if args.LatexLines!= None:
+	for line in args.LatexLines:
+		text_file.write("\tCanv_Latex_Line = '%s';\n"%(line))
+	pass
+	pass
```

Comment:
tabs not aligned correctly. If the first `pass` statement is supposed to belong to this for loop then additional tabs are needed

Reply:
fixed..